lib/src/kbasesearchengine/search/ElasticIndexingStorage.java

```diff
@@ -1320,7 +1325,16 @@ public FoundHits searchObjects(String objectType, MatchFilter matchFilter,
     private List<Map<String, Object>> prepareMatchFilters(MatchFilter matchFilter) {
         List<Map<String, Object>> ret = new ArrayList<>();
         if (matchFilter.fullTextInAll != null) {
-            ret.add(createFilter("match", "_all", matchFilter.fullTextInAll));
```

Comment:
Also, do all queries now use AND logic rather than OR, and so have consistent behavior?

Reply:
yes. Most methods that generates a call to elasticsearch calls prepareMatchFilters. The exceptions are the indexing methods (which I don't need to change) and getObjectsByIds. I think that is the only place that should have the old logic (OR) b/c of the nature of that function which allows you to pass in multiple ids....